bindata/manifests/daemon/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ingress-node-firewall-daemon
+  namespace: '{{.NameSpace}}'
+spec:
+  selector:
+    matchLabels:
+      app: ingress-node-firewall-daemon
+  template:
+    metadata:
+      labels:
+        app: ingress-node-firewall-daemon
+        component: network
+        type: infra
+    spec:
+      containers:
+        - command: ['/bin/sh', '-c', 'mount bpffs /sys/fs/bpf -t bpf && daemon']
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          image: '{{.Image}}'
+          name: daemon
+          securityContext:
+            privileged: true
+            runAsUser: 0
+            capabilities:
+              add:
+                - CAP_BPF
+                - CAP_NET_ADMIN
```

Comment:
can you add this as well so we can see the last 80 lines of the log when a pod errors out:
```yaml
terminationMessagePolicy: FallbackToLogsOnError
```